lib/improver/wind_direction.py

```diff
@@ -80,16 +80,50 @@ class WindDirection(object):
     Step 6 still needs more development so it is only included in the code
     as a placeholder.
 
+    Keyword Args:
+        backup_method (str):
+            Backup method to use if the complex numbers approach has low
+            confidence.
+            "first realization" (default) uses the value of realization zero.
+            "neighbourhood" recalculates using the complex numbers approach
+            with additional realizations extracted from neighbouring grid
+            points from all available realizations.
+
     """
 
-    def __init__(self):
+    def __init__(self, backup_method='first realization'):
         """Initialise class."""
-        pass
+        self.backup_methods = ['first realization', 'neighbourhood']
+        self.backup_method = backup_method
+        if self.backup_method not in self.backup_methods:
+            msg = ('Invalid option for keyword backup_method ' +
```

Comment:
Doesn't need the '+' here, line continuation will handle this.

Reply:
Thanks. I didn't know that - change on the way.